app/actions/games.js

```diff
@@ -0,0 +1,21 @@
+import { Config }       from '../../config';
+import { API }          from '../utils/api';
+import { checkVersion } from './utils';
+
+export const SET_GAMES  = 'SET_GAMES';
+
+export function listGames () {
+  return (dispatch) => {
+    dispatch(checkVersion());
+
+    return API.get(`${Config.API_HOST}/games`)
+    .then((games) => {
+      dispatch(setGames(games));
+      return games;
```

Comment:
since we're setting `games` on the global state (with the previous line), we dont need to return them directly; the components will get it from the state, not from the result of this promise